### DISCUSSION TITLE
Many messages at startup

### DISCUSSION THREAD

**Discussion by aprelvovanya** [hyperjumptech/monika#931] [Q&A] [answered]
Monika >=1.13.0

I have 12 probes configured.
When I start it, I immediately receive messages in the console:

INFO: 2022-10-21T19:12:20.754Z 0 id:3 200 GET https://.../ 34ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:20.909Z 0 id:10 200 GET https://.../ 189ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:20.919Z 0 id:1 200 GET https://.../ 208ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:20.952Z 0 id:7 200 GET https://.../ 236ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:21.079Z 0 id:2 200 GET https://.../ 364ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:21.848Z 0 id:6 200 GET https://.../ 138ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:22.065Z 0 id:9 200 GET https://.../ 353ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:22.813Z 0 id:4 200 GET https://.../ 105ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:22.870Z 0 id:0 200 GET https://.../ 165ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:22.931Z 0 id:5 200 GET https://.../ 221ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:22.940Z 0 id:11 200 GET https://.../ 227ms, NOTIF: service is back up
INFO: 2022-10-21T19:12:23.222Z 0 id:8 200 GET https://.../ 510ms, NOTIF: service is back up

As well as 24 telegram messages (Message: Response status is 200, Message: Response time is 353ms)
 
Monika <= 1.12.1 everything works fine

Linux 5.15.0-1021-azure 26-Ubuntu GNU/Linux and Win10
node v18.10.0

**Comment by aprelvovanya** [comment]
https://github.com/hyperjumptech/monika/pull/934

**Comment by sapiderman** [comment, marked as answer]
> Monika >=1.13.0
> 
> I have 12 probes configured. When I start it, I immediately receive messages in the console:
> 
> 
> As well as 24 telegram messages (Message: Response status is 200, Message: Response time is 353ms)
> 
> Monika <= 1.12.1 everything works fine
> 
> Linux 5.15.0-1021-azure 26-Ubuntu GNU/Linux and Win10 node v18.10.0

Hi @aprelvovanya,  We fixed it and it has merged on ticket #934.
This was a bug in the state engine when monika first starts. Should be fine now. Let us know if you have any feedback!
